android/app/build.gradle

```diff
@@ -0,0 +1,77 @@
+def localProperties = new Properties()

```

Comment:
Shouldn't these be in example app instead of android wrapper?

Reply:
these are in the example /android/app/build.gradle